python/ray/actor.py

```diff
@@ -479,7 +484,8 @@ def options(self,
                 placement_group="default",
                 placement_group_bundle_index=-1,
                 placement_group_capture_child_tasks=None,
-                runtime_env=None):
+                runtime_env=None,
+                scheduling_strategy=None):
```

Comment:
Update pydoc for scheduling_strategy, and comment that placement_group* are deprecated?

Reply:
We should also make a (maybe another separate) PR refactoring internal code to not use placement_group, and updating the docs.